balena/models/os.py

```diff
@@ -493,6 +494,89 @@ def is_architecture_compatible_with(self, os_architecture: str, application_arch
 
         return True
 
+    def get_supervisor_releases_for_cpu_architecture(
+        self, cpu_architecture_slug_ord_id: Union[str, int], options: AnyObject = {}
+    ) -> List[ReleaseType]:
+        """
+        Returns the Releases of the supervisor for the CPU Architecture
+
+        Args:
+            cpu_architecture_slug_ord_id (Union[str, int]): The slug (string) or id (number) for the CPU Architecture.
+            options (AnyObject): extra pine options to use.
+
+        Returns:
+            ReleaseType: release info.
+
+
+        Example:
+            results = balena.models.os.get_supervisor_releases_for_cpu_architecture('aarch64');
+            results = balena.models.os.get_supervisor_releases_for_cpu_architecture(
+                'aarch64',
+                { $filter: { raw_version: '12.11.0' } },
+            );
+        """
+        return self.__pine.get(
+            {
+                "resource": "release",
+                "options": merge(
+                    {
+                        "$select": ["id", "raw_version", "known_issue_list"],
+                        "$filter": {
+                            "status": "success",
+                            "is_final": True,
+                            "is_invalidated": False,
+                            "semver_major": {"$gt": 0},
+                            "belongs_to__application": {
+                                "$any": {
+                                    "$alias": "a",
```

Comment:
Not sure what the standard is but everywhere else I've seen we use the first letters of each word in the property for best understanding, i.e. `bta` for `belongs_to__application`, not just `a`. Not sure if this is just something Thodoris has instilled in me though rather than proper convention :P

Same for the ones below of course. Ultimately up to you of course, the goal is just for it to be understandable

Reply:
Just keeping parity with nodejs sdk query which I use always as source of truth: https://github.com/balena-io/balena-sdk/blob/88070df6f13c21db0c6f83cc1683f46a1a8f4660/src/models/os.ts#L1009

I think the reasoning behind here is that this query is already huge and the expanded resource is just an `a`application